packages/@glimmer/runtime/lib/vm/serialize-builder.ts

```diff
@@ -0,0 +1,23 @@
+import { NewElementBuilder, ElementBuilder } from "./element-builder";
+
+import Bounds, { bounds, currentNode } from '../bounds';
+import { Simple } from "@glimmer/interfaces";
+
+export class SerializeBuilder extends NewElementBuilder implements ElementBuilder {
+  __appendHTML(html: string): Bounds {
+    let first = this.__appendComment('%glimmer%');
+    super.__appendHTML(html);
+    let last = this.__appendComment('%glimmer%');
```

Comment:
This probably wants to be a matching pair, like `<!-- {{{ -->` and `<!-- }}} -->` perhaps. If we think it's not important to have a matching pair for this case then an empty comment is just fine.